mrt_file_server/views.py

```diff
@@ -100,11 +100,21 @@ def download_schematic_post():
     log_warn('SCHEMATIC_DOWNLOAD_FILENAME_WHITESPACE', filename)
     return
 
-  filename = "{}.schematic".format(secure_filename(filename))
-
-  if file_exists_in_dir(downloads_dir, filename):
-    log_info('SCHEMATIC_DOWNLOAD_SUCCESS', filename)
-    return send_from_directory(downloads_dir, filename, as_attachment = True)
+  filename1 = "{}.schematic".format(secure_filename(filename))
+  filename2 = "{}.schem".format(secure_filename(filename))
+
+  file1_exists = file_exists_in_dir(downloads_dir, filename1)
+  file2_exists = file_exists_in_dir(downloads_dir, filename2)
```

Comment:
Variable names like `filename1` and `filename2` are not a good idea and should be more descriptive of what they do.